utils/getInvoices/tests/getInvoices.electricity.test.ts

```diff
@@ -6,143 +6,551 @@ import { ServiceType } from '@utils/constants'
+      const service: Partial<IService> = {
+        electricityPrice: 10,
+      }
+      const company: Partial<IRealestate> = {
+        cleaning: 10,
+      }
+      const prevPayment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Cleaning,
+            amount: 12,
+            lastAmount: 1,
+            price: 10,
+            sum: 10,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+        prevPayment,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Electricity,
+        amount: 0,
+        lastAmount: 0,
+        price: 10,
+        sum: 0,
+      })
+    })
   })
 
-  it('should NOT load Electricity without props', () => {
-    const company: Partial<IRealestate> = {}
-    const service: Partial<IService> = {}
-    const payment: Partial<IPayment> = {}
-    const prevPayment: Partial<IPayment> = {}
+  describe('props: { service, company, payment } with prio to payment', () => {
+    it('should NOT load when service = null, company = null, payment = null', () => {
+      const service: Partial<IService> = null
+      const company: Partial<IRealestate> = null
+      const payment: Partial<IPayment> = null
+
+      const invoices = getInvoices({
+        service,
+        company,
+        payment,
+      })
 
-    const invoices = getInvoices({
-      company,
-      service,
-      payment,
-      prevPayment,
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
     })
+    it('should load when service = null, company = null, payment = { invoice: [Electricity] }', () => {
+      const service: Partial<IService> = null
+      const company: Partial<IRealestate> = null
+      const payment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Electricity,
+            amount: 12,
+            lastAmount: 1,
+            price: 10,
+            sum: 10,
+          },
+        ],
+      }
 
-    expect(invoices).not.toContainEqual(
-      expect.objectContaining({ type: ServiceType.Electricity })
-    )
+      const invoices = getInvoices({
+        service,
+        company,
+        payment,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Electricity,
+        amount: 12,
+        lastAmount: 1,
+        price: 10,
+        sum: 10,
+      })
+    })
+    it('should load when service = { electricityPrice: 10 }, company = null, payment = { invoice: [Electricity] }', () => {
+      const service: Partial<IService> = {
+        electricityPrice: 10,
+      }
+      const company: Partial<IRealestate> = null
+      const payment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Electricity,
+            amount: 12,
+            lastAmount: 1,
+            price: 10,
```

Comment:
It's betterr NOT to use same `electricityPrice` and `invoice.electricity.price` to test out the response value. For example if:
```
const service = {
  electricityPrice: 20,
}
const payment = {
  invoice: [
    {
      type: ServiceType.Electricity,
      price: 10,
      ...    
     }
  ]
}
```

Then the expected result is:
```
{
  type: ServiceType.Electricity,
  price: 10,
  ...    
}
```

In this case we make sure that the `price` value was received from `payment` and NOT form `service`